src/main/java/factionsplusplus/commands/HelpCommand.java

```diff
@@ -22,6 +24,7 @@
 import factionsplusplus.builders.ArgumentBuilder;
 
 import java.util.ArrayList;
+import java.util.Arrays;
```

Comment:
Is this used?

Reply:
https://github.com/nortoh/FactionsPlusPlus/pull/104/files/45b5ee2310c9ea435d4e57c0fe88308c90b806ed#diff-ca817c541a5d857723f7044d7bd0d1404193675b90e470e6e0103296f28965ffR139